src/System.Runtime.Numerics/tests/BigInteger/modpow.cs

```diff
@@ -161,6 +161,23 @@ public static void ModPow2Large1SmallInt()
         }
 
         [Fact]
+        public static void ModPow3LargeInt()
+        {
+            byte[] tempByteArray1 = new byte[0];
+            byte[] tempByteArray2 = new byte[0];
+            byte[] tempByteArray3 = new byte[0];
```

Comment:
Thanks; I didn't look at the rest of the file and thus didn't realize this was copied from elsewhere... please feel free to fix those up as well ;)